Integrations/integration-Guidance_Encase_Endpoint.yml

```diff
@@ -0,0 +1,238 @@
lVb91dYWNa9iwaPIQCBpwn0ZBdI6WzX/Sqde163TOR9IxAtmPubinbfhXIgBqQ5/hOYV2z34kA7BBIl0JP7XoT7qYzZyxguLwhAAALbE+htLwIJXSagW/NaXqki9+iXMqFy3OU2xHcIbErA4tbcMnn6i0zuDDb1uRP1rhZL2Jpg9mHeibbBSQhAAAI1E6gm2DXjYB4BwTJWRpbgVS+Rq0VANgJ3ei5MV/WLco8TkKD5E0v9d/1tkv76weNecRYC3SKg32Mf3LBbXuOtBQLVGLEs5LFg61M2LgKDp0+V43x8AvocUGVIz6S/Ep8YGiAAAQisSKCME2wkfbj9FS2g2GYETjarZqOWzN8l3M8JIxsUsRICEGiUQBnnvidBfF8b9QNlEIBAtgQICs62aZtzrOfDcXPa6tdUBQlKppX6JSMRAvkTkMB6cxM+Mrkz1UCY/FtnfQ+Hb98PdbHE+jVbrXGuE+OtWoByCEAAAgYJ6AR72ClOM9qCnCzB61yHPxTjdYq3WFbv89MW9Wen2lz/PXgr12p21woOpUtAgy9lIczP6VqIZV0lINnrJrn4roHBP705OmH8qL0WvRqj8vOt+yv94rQ9L9AMAQhAIFkCscbT95L1OA/DYrVbKnT2hn/9234qxlzbEf6yP5L3thLpXBvPXwhAICqB0scJCpbsJ7n/3kdtF4RDAALfCBAU/I0F7zYksJikPduwehLVJDD4owRGjJIwBiMgYIhAz7u5IXOvTd1z/eL4+gN/rwjoYEsI9jLxeWfyGuSygwAEINAqAcmaP5bn3/8RI7IZ0CZL8OqXVJUl2Mi2h9Ku09U9o+RKBC52ZlLufKWyCRSSoKwhAVkJNAQmJEdAx+IkY/CfyRmGQZ0mkFO2YG1IDUbVLLXch5q/rGWc/ljGqP5LNG/dX5n98etp8x6gEQIQgEB3CaQY1JlNaxT9/IPEiv4wvfYiS3B6bYJFEEiDgMxR76dhCVZAAAIQWE6AoODlXDi6JoEcJuEJDF6z0SkOASVQ2gzIrLaCZCHAzTWsE1xl0dcAka0nW26ENvUm2LwGm8KDHghAAAK3Cejv/fDwaCb3wQ+3j1t/H4L7QjbZNVpx50InWKzc86dreEbRFQjMjl9/lUBCfe6z8tqVBX0jK8ZiJwSaJCABmCOLu/c0yQhdzRPIKVuw0tPAYN1dQ3fZaJ5m9zRqIJlmaNad/brnPR5DAAIQaI6ALuSJpq3o70eT3XHB2p8XBGc5YyidG6Xkn2QJ3nfBvUjJJmyBAASSIhAnQ77R+IukWgZjIACBigBBwVwItRDIIVuwgiAwuJbLASEdIrAYPDKTaex201Tf9zdHg9vHOvteMidvvR1jW/D6haWglrYooRcCEICAk+zAAw1okOChl7nhkE7tODefYvojz0CTmPLrkr0I9j6pSx5yviMw/e5T4h9kIQPf8cTbCPPaI9C76A/197I9C9AMge8J5JYteOHdrjw/zdll7vu2rvtTtZvJZf+k5vGprAOn6m4D5EEAAhCoicBBTXIQs4SALPKdLzmczSF9Dkgq27SRnbayuQBwBAKGCLBw1FBjYSoEOkyAoOAON37drnvvs5ioIzC47isDebkT8M7uIIQEGMy6/tAuky5TzZxs8TqtgoXYBtJi02EzBCDQIIFb2YH/IWqtZIddmZA8h3yKmuFmZUtsFFwEs8TJYFAzgp4PxzWLRNyCwOI7YylIZm/49v2QBoQABO4T0Gxh8ns5kjMmF+ve94gjORDILVvwok00MPijjqHo83UO7ZSKD1V24MOj+WI3k1r7K7Ig8jQVP7EDAhCAQGoE5DfycxSbeo77ZBSwN0JPbt7l+qbop9P/DyEdW3Jtb/yCgFUCwUe73zHfYfWiwG4IpEeAoOD02sSsRbN3b2bROpENU7ke5G1YLeogYJRAmBk1XM2+zvZyYNiHjU23HBCsTsvk93xj56kIAQhAoAMEqmxbmWYHXjTfuXtWjDvQlLW5WDo/qk1YXEHn7mLH8jNmXDo1SJfsQsc1iGlMRBkC3/XGaKPIGgHZvetEvtNDa3Zjb74EdAJTrsk/c/RQF1WX/UKzBh/k6F/TPlX9Fc0O7NiWu2n26IMABCAQjUBw3COjwRXBHZgTkeetJPr/4T/3h847EwvrY15yyIYABB4g0HODB85wGAIQgEAyBAgKTqYp8jAkp0wQOsj7k2QpIPtDHtcmXkQkYD9go5OBwdYDghdX9DTilY1oCEAAAmYJaEbNl4dHpzGybaUERbYUPJ6RMX7lJpF7/8BKwIUEEs00++XKzlFwfQJFf7p+pfZq6LWb1Bai7aFAMwSWElgEYb5eepKDEGiDQL+YiNosM1jrttaSUOJ/hm+P1EdeGxDQ59Kf3hydxO6vyOQXz5MbtA9VIACBbhDo+Tg7QJbB7XeDYDte6oJA0ZzlM9YtontJLMDqlcNbNvEWAhCAwPcEQnj+/YF6PuWShLEeGkiBAAS2JUBQ8LYEqf8dgdwyQejEY9gpTpLofHxHmg8QSIeABmzId+VTOhZtZMl1YPBoo9qGKlXbyMt2l7rwwZDZy0w9WwyALTvHMQhAAAKdJFBNruvWuyH8lwDIPZPF2ezd4aSTDb2506PNqzZc04fjhjV2Tl31DG8ti+Pls3HnGgqHIbAGARmTO841O+saGCiaCAFduKULuBIxJ4oZIbjfdCGePoNHUZChUF3gI7x0TOofGlwd3UXvTqLrQAEEIAABqwTKOAsnGvl9t8q8JrtlPm5ek6h0xQTffv8/+GG6gLAMAhBom4DsyHcQwwYWNsagikwIdJcAQcHdbft4nueXCWKvyv7w5qj9Dki8VkMyBLYkEGZbCkihugYGf8w504tOvug2lxkEBMsuWXlPLqbwhcAGCEDADgFZwDZaZAb+hy5qs2P55pbKfWC0ee3u1dRnACv3fwmw+cLCn8au0WljmmpQJM/qI3byqQEkIrImIIHBI/0dzdpJnLND4J/9YzH2zI7BG1m6pwGuGuhKR
```

Comment:
Because the args are different and the template is different, don't think it's so complicated as it is now.